src/helpers/queryHelpers.ts

```diff
@@ -60,10 +60,13 @@ export const useServiceQuery = <TItem, TColumnKey extends string, TSortableColum
   columnOrderMap?: Record<TSortableColumnKey, string>;
   tableState: TableState<TItem, TColumnKey, TSortableColumnKey>;
   setRefreshTime?: (date: Date) => void;
-}) =>
-  useQuery<ServiceQueryResult<TItem>>({
+}) => {
+  const pollInterval = Number(process.env.REACT_APP_POLL_INTERVAL) || 60000;
```

Comment:
required:
we should move to the specific method instead of the generic handling of `Number`. what you have works but can lead to subtle JS behavior alterations. 

```
Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10)
```

optional:
instead of a hardcoded value we could also make the value `undefined`. It'll default towards whatever was there originally. and coincidentally would restore the exact same handling of this value as `v1.8` ... https://github.com/quipucords/quipucords-ui/blob/1.8.0/src/common/helpers.js#L436

Plus it's one less thing to track down for future us

```
Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10) || undefined
```

Reply:
This didn't really work, because Number.parseInt expects a string, so it complains beause it might be undefined. 
`  Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10) || undefined`
As an alternative I check it it exists first, then try to parse it, what do ya think ?
```
  const pollInterval = process.env.REACT_APP_POLL_INTERVAL 
  ? Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10) 
  : undefined;
```